src/components/DeckBuilderRNGField/index.js

```diff
@@ -67,6 +70,11 @@ const DeckBuilderRNGField = props => {
       >
         Friendly
         <span className='DeckBuilderRNGField__radio-info'>
+          {freezeCard ? (
+            <span key={freezeCard}>
```

Comment:
```suggestion
            <span>
```

The `key` is only necessary in loops. Ref: https://reactjs.org/docs/lists-and-keys.html.